Demo/build.gradle

```diff
@@ -7,7 +7,7 @@ android {
     defaultConfig {
         applicationId "com.schedjoules.demo"
         minSdkVersion 14
-        targetSdkVersion 22
+        targetSdkVersion 23
```

Comment:
Note that this change has implication on how permissions are handled. In particular this requires the app to request permissions at runtime, otherwise calendar synchronization won't work.

Reply:
Oh, that sounds like something important. Shall we create a ticket to track it?

Reply:
yes, please.

Reply:
added: https://github.com/schedjoules/android-public-calendars-sdk/issues/10